include/GMRESSolver.hpp

```diff
@@ -0,0 +1,352 @@
+#pragma once
+#include "LinAlgHelpers.hpp"
+#include "Types.hpp"
+#include <algorithm>
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <ostream>
+#include <sstream>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace blitzdg {
+namespace details {
```

Comment:
I think I'd like to see namespaces start a new level of indentation.

Reply:
Sure thing bud.